acados/ocp_nlp/ocp_nlp_sqp.h

```diff
@@ -77,7 +79,10 @@ typedef struct
     ocp_nlp_memory *nlp_mem;
 
     int sqp_iter;
-    int dummy;  // NOTE(dimitris): make struct size multiple of 8
```

Comment:
Is this not relevant any more? I vaguely remember that we should have an even number of ints in our structs to make sure that we are 8-byte aligned every += sizeof(struct) at the beginning of the assign routines.

Reply:
Good point. Don't we anyway always align to 8 bytes ?

Reply:
I think not every +sizeof(struct), coz it would be a bit tedious, but I haven't checked that since quite a while now..